Classes/Core/Testbase.php

```diff
@@ -14,7 +14,6 @@
  * The TYPO3 project - inspiring people to share!
  */
 
-use Doctrine\DBAL\DBALException;
```

Comment:
If we used this …:

```php
Doctrine\DBAL\Exception as DBALException;
```

… in all affected classes, we can leave the usages as-is, and still benefit from the short class names. What do you think?

Reply:
Done.